src/jon_mikkel_ex/ex02/file_stats.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+__author__ = "Jon-Mikkel Korsvik"
+__email__ = "jonkors@nmbu.no"
+
+
+def char_counts(text_filename):
+    """ Opens a file for reading and makes a list of each character with a
+        code lower than 256, and records the occurrences
+        Returns
+        --------
+        A list of each characters code and its occurrences
+    """
+
+    list_of_char_code_with_freq = [0 for _ in range(256)]
+    with open(text_filename, 'r', encoding='utf-8') as txt:
```

Comment:
To avoid using the nested for-loop just assign the file to `text` or something single string using `txt.read()`